ui/apos/components/AposDuplicateImportModal.vue

```diff
@@ -323,6 +325,15 @@ export default {
   display: flex;
   flex-direction: column;
   align-items: baseline;
+  max-height: 50vh;
+  overflow-y: auto;
+}
+
+.apos-import-duplicate__section thead {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  backgroud-color: var(--a-base-10);
```

Comment:
```suggestion
  position: sticky;
  top: 0;
  background-color: var(--a-background-primary);
```

- `sticky`, good stuff.
- no need for z-index it appears
- please change background color to white (`var(--a-background-primary)`) and fix typo

Reply:
DOne!